edge-util/src/Microsoft.Azure.Devices.Edge.Util/Logger.cs

```diff
@@ -43,23 +43,52 @@ static ILoggerFactory GetLoggerFactory()
             Serilog.Core.Logger loggerConfig = new LoggerConfiguration()
                 .MinimumLevel.ControlledBy(levelSwitch)
                 .Enrich.FromLogContext()
+                .Enrich.With(SeverityEnricher.Instance)
                 .WriteTo.Console(
-                    outputTemplate: "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level:u3}] - {Message}{NewLine}{Exception}").CreateLogger();
+                    outputTemplate: "<{Severity}> {Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level:u3}] - {Message}{NewLine}{Exception}").CreateLogger();
 
             if (levelSwitch.MinimumLevel <= LogEventLevel.Debug)
             {
                 // Overwrite with richer content if less then debug
                 loggerConfig = new LoggerConfiguration()
                     .MinimumLevel.ControlledBy(levelSwitch)
                     .Enrich.FromLogContext()
+                    .Enrich.With(SeverityEnricher.Instance)
                     .WriteTo.Console(
-                        outputTemplate: "{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level:u3}] [{SourceContext:1}] - {Message}{NewLine}{Exception}").CreateLogger();
+                        outputTemplate: "<{Severity}> {Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level:u3}] [{SourceContext:1}] - {Message}{NewLine}{Exception}").CreateLogger();
             }
 
             ILoggerFactory factory = new LoggerFactory()
                 .AddSerilog(loggerConfig);
 
             return factory;
         }
+
+        // This maps the Edge log level to the severity level based on Syslog severity levels.
+        // https://en.wikipedia.org/wiki/Syslog#Severity_level
+        // This allows tools to parse the severity level from the log text and use it to enhance the log
+        // For example errors can show up as red
+        class SeverityEnricher : ILogEventEnricher
+        {
+            static readonly IDictionary<LogEventLevel, int> LogLevelSeverityMap = new Dictionary<LogEventLevel, int>
+            {
+                [LogEventLevel.Fatal] = 0,
+                [LogEventLevel.Error] = 3,
+                [LogEventLevel.Warning] = 4,
+                [LogEventLevel.Information] = 6,
+                [LogEventLevel.Debug] = 7,
+                [LogEventLevel.Verbose] = 7
```

Comment:
I'm assuming this conversion matches the rust conversion?

Reply:
Yep - fn syslog_level(level: Level) -> i8 {
    match level {
        Level::Error => 3,
        Level::Warn => 4,
        Level::Info => 6,
        Level::Debug | Level::Trace => 7,
    }

https://github.com/Azure/iotedge/blob/master/edgelet/iotedged/src/logging.rs#L72